receipt_label/receipt_label/langchain/currency_validation.py

```
@@ -0,0 +1,653 @@
+    """Final node: Combine all results and calculate final metrics.
+
+    Args:
+        state: Current workflow state
+        save_dev_state: Whether to save state for development (default: False)
+    """
+
+    print(f"🔄 Combining results")
+
+    # 💾 Save state for development if requested (align with ./dev.states)
+    if save_dev_state:
+        print(f"   💾 Saving state for development...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)